packages/itwin/tree-widget/src/components/trees/category-tree/CategoriesTree.tsx

```diff
@@ -116,6 +128,7 @@ export function CategoryTree(props: CategoryTreeProps) {
       iconsEnabled: false,
       descriptionEnabled: true,
       levelOffset: 10,
+      onVisibilityToggled: () => reportUsage?.({ featureId: "visibility-change", reportInteraction: true }),
```

Comment:
Looks like `useFeatureReporting` does not return `reportUsage` as `undefined` anymore so `?.` can be removed in all the places.

```suggestion
      onVisibilityToggled: () => reportUsage({ featureId: "visibility-change", reportInteraction: true }),
```